source/plot/SvgPlot.ooc

```diff
@@ -118,17 +114,17 @@ SvgPlot: class {
 	}
 
 	fillColorList: func {
-		this colorList = VectorList<String> new()
-		this colorList add("crimson")
-		this colorList add("black")
-		this colorList add("darkmagenta")
-		this colorList add("seagreen")
-		this colorList add("dodgerblue")
-		this colorList add("darkorange")
-		this colorList add("hotpink")
-		this colorList add("lightgreen")
-		this colorList add("yellowgreen")
-		this colorList add("red")
+		this colorList = VectorList<ColorBgra> new(10, false)
+		this colorList add(ColorBgra new(100, 0, 0, 255))
+		this colorList add(ColorBgra new(0, 0, 0, 255))
+		this colorList add(ColorBgra new(50, 0, 50, 255))
+		this colorList add(ColorBgra new(0, 50, 0, 255))
+		this colorList add(ColorBgra new(0, 20, 40, 255))
+		this colorList add(ColorBgra new(100, 50, 0, 255))
+		this colorList add(ColorBgra new(200, 0, 100, 255))
+		this colorList add(ColorBgra new(0, 255, 0, 255))
+		this colorList add(ColorBgra new(128, 255, 0, 255))
+		this colorList add(ColorBgra new(0, 200, 0, 255))
```

Comment:
The names are specific to SVG and had to be abstracted into the SVG folder. I also need the color values to extract alpha for opacity.